solutions/bronze/usaco-1251.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: usaco-1251
+source: USACO Bronze 2022 December
+title: Cow College
+author: Chu Minjae
+---
+[Official Analysis (C++, Java, Python)](http://www.usaco.org/current/data/sol_prob1_bronze_dec22.html)
+
+## Explanation
+
+Basically, the solution is using sorting and for loop to find the max tuition. I will explain it one by one. First, the type should be assigned as long long since the max tuition and the best tuition on that specific point
+can exceed the range of -2,147,483,648 to 2,147,483,647. The number of cows can be 1 to 100,000, while the maximum tuition varies from 1 to 1,000,000. Multiplying both numbers exceeds the range, so the array and variables that will
+hold the maxTuition and bestTuition on that specific point should be assigned with long long rather than int form. After assigning the variables, the number of cows and the maximum tuition that each cow is willing to pay
```

Comment:
variables should have markup applies (either latex or inline code)

Reply:
Solve it :)